clients/drcachesim/tools/record_filter_launcher.cpp

```diff
@@ -125,6 +133,10 @@ _tmain(int argc, const TCHAR *targv[])
 {
     disable_popups();
 
+#if defined(WINDOWS) && !defined(_UNICODE)
```

Comment:
Do we need this? Seems redundant to have a check like this when the fix is just another #define.

Reply:
I want to avoid someone else having to spend time tracking down a weird error like I had to do, in the case that maybe these defines get shuffled down below some earlier #include of the headers that need the defines.  So IMHO it is worth having this check down here.